src/medias/images/UGT_Asset_UI_Back.tsx

```diff
@@ -10,7 +10,7 @@ export const ImgBack: React.FunctionComponent<ImgBackProps> = ({ alt, ...props }
       {Boolean(alt) && <title>{alt}</title>}
       <path
         d="M12.3588 4.1438L13.8109 5.60447L8.06434 11.3954H20.5977V13.4672H8.06434L13.8109 19.2581L12.3588 20.7188L4.12 12.4313L12.3588 4.1438Z"
-        fill="black"
+        fill="var(--color-action)"
```

Comment:
Maybe we wan't this to be passed by a prop, similar to `src/medias/images/UGT_Asset_UI_ButtonNext.tsx`